core/treepath/treepath.go

```diff
@@ -177,3 +177,16 @@ func (tp Treepath) Apply(n *game.Node) *game.Node {
 	}
 	return curNode
 }
+
+// Returns the treepath as a string. 
+// examples: 
+// 	[]				becomes "[]"
+// 	[1]				becomes	"[1]"
+// 	[1,2,0,2,2,2] 	becomes "[1,2,0,2,2,2]"
+func (tp Treepath) String() string {
```

Comment:
Note: I think we want to string methods.

`String()`, which is this method and `CompactString()` which returns the shortened version with the `:` character. Ex: `CompactString([0,0,0,0]) => "0:4"`

I would do that as a follow-up PR.